ocaml/xapi/xapi_host.ml

```diff
@@ -1400,7 +1400,7 @@ let apply_edition ~__context ~host ~edition ~force =
     raise (Api_errors.Server_error (Api_errors.ha_is_enabled, []))
   else
     let additional = if force then ["force", "true"] else [] in
-    apply_edition_internal ~__context ~host ~edition ~additional
+    apply_edition_internal ~__context ~host ~edition:edition ~additional
```

Comment:
That shouldn't be needed.

Reply:
Whoops, holdover from removing `edition_str`.